### DISCUSSION TITLE
Is it possible to send logs to Jaeger?

### DISCUSSION THREAD

**Discussion by andreivaloshyn** [grafana/tempo#1779] [Q&A] [answered]
Hello,
I'm using tempo-distributed helm chart to deploy tempo into EKS.
Is there a way to set up tempo to do something like `remote_write` to Jaeger endpoint to have traces across both systems - tempo  and jaeger?
I can't find anything related to my question in Config section https://grafana.com/docs/tempo/latest/configuration/

**Comment by mapno** [comment, marked as answer]
Hi @andreivaloshyn, you mean duplicating data to Jaeger? Unfortunately that's not possible. For that, the best option will be using a tracing pipeline like the [OTel collector](https://github.com/open-telemetry/opentelemetry-collector) or the [Grafana Agent](https://github.com/grafana/agent), which are designed for that specific need (among others).

**Reply by andreivaloshyn** [reply]
Hey @mapno, thanks for your reply.
yes, I meant data duplicating.

**Reply by ovidiubuligan** [reply]
@andreivaloshyn We also use OtelCcollector as our main ingester so we can distribute to different syncs (with different sampling strategies ). Also you can go full otel with jaeger since it supports receiving OTLP traces  (so basically forget about specific jaeger protocol ) : https://www.jaegertracing.io/docs/1.39/client-libraries/
